src/Analyzers/CSharp/CodeFixes/UseCompoundAssignment/CSharpUseCompoundAssignmentCodeFixProvider.cs

```diff
@@ -47,6 +48,8 @@ private static ExpressionSyntax Postfix(SyntaxKind kind, ExpressionSyntax operan
         private static ExpressionSyntax Prefix(SyntaxKind kind, ExpressionSyntax operand)
             => SyntaxFactory.PrefixUnaryExpression(kind, operand);
 
+        protected override SyntaxTriviaList PrepareRightExpressionLeadingTrivia(SyntaxTriviaList initialTrivia) => initialTrivia.SkipWhile(el => el.Kind() is SyntaxKind.WhitespaceTrivia or SyntaxKind.EndOfLineTrivia).ToSyntaxTriviaList();
```

Comment:
You could use this [`IsKind()`](https://sourceroslyn.io/#Microsoft.CodeAnalysis.CSharp.Workspaces/J/s/src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Extensions/SyntaxNodeExtensions.cs/SyntaxNodeExtensions.cs,70) overload instead

Reply:
There is only 1 overload for `SyntaxTrivia`, wich is a type of `el` here, that takes only 1 `SyntaxKind`. No overloads with 2 `SyntaxKind`s for comparing exist.

Reply:
Oops, my bad. [This ](https://sourceroslyn.io/#Microsoft.CodeAnalysis.CSharp.Workspaces/SyntaxTriviaExtensions.cs,24)is what I meant to say. And I just noticed there's already an [`IsWhitespaceOrEndOfLine`]([IsWhitespaceOrEndOfLine](https://sourceroslyn.io/Microsoft.CodeAnalysis.CSharp.Workspaces/R/07abdeb1515ceffd.html)) helper